src/main/java/club/minnced/discord/webhook/WebhookClient.java

```diff
@@ -669,11 +669,11 @@ else if (!response.isSuccessful()) {
                           response.code(), new IOUtil.Lazy(() -> new String(IOUtil.readAllBytes(IOUtil.getBody(response)))));
                 return;
             }
-            remainingUses = Integer.parseInt(response.header("X-RateLimit-Remaining"));
-            limit = Integer.parseInt(response.header("X-RateLimit-Limit"));
+            remainingUses = Integer.parseInt(response.header("X-RateLimit-Remaining","50"));
+            limit = Integer.parseInt(response.header("X-RateLimit-Limit","50"));
```

Comment:
Instead of just defining a different value this should be using `Retry-After` header if present. If no header is available we should simply not update the bucket at all.